satpy/enhancements/atmosphere.py

```diff
@@ -22,12 +22,34 @@
 
 
 def essl_moisture(img, low=1.1, high=1.6) -> None:
-    """Low level moisture by European Severe Storms Laboratory (ESSL).
+    r"""Low level moisture by European Severe Storms Laboratory (ESSL).
 
-    Must be passed exactly two projectables.  The first one should correspond
-    to a channel at around 0.86 µm, the second one at 0.91 µm.
+    Expects a mode L image with data corresponding to the ratio of the
+    calibrated reflectancys for the 0.86 µm and 0.906 µm channel.
```

Comment:
Reflectances? Reflectivities?

Reply:
Fixed in https://github.com/pytroll/satpy/pull/2164/commits/11afd9dcfe5cd844b1e48ba18f8a63c1a93dfd88